lib/modules/datasource/aws-machine-image/readme.md

```diff
@@ -1,6 +1,30 @@
 > :warning: **This datasource is _experimental_**: Be aware that its syntax and behavior may change at any time!
 
-This datasource returns the latest [Amazon Machine Image](https://docs.aws.amazon.com/en_en/AWSEC2/latest/UserGuide/AMIs.html) via the AWS API (valid credentials required).
+This datasource returns the latest [Amazon Machine Image](https://docs.aws.amazon.com/en_en/AWSEC2/latest/UserGuide/AMIs.html) via the AWS API.
+
+As this datasource relies on the AWS-SDK for JavaScript, the configuration looks similar to other AWS Tools and the common configuration options are supported, for example:
+
+- Setting the region via `AWS_REGION` (environment variable) or your `~/.aws/config` file
+- Provide credentials via `AWS_ACCESS_KEY_ID`and `AWS_SECRET_ACCESS_KEY` (environment variable) or your `~/.aws/credentials` file
+- Select the profile to use via `AWS_PROFILE` environment variable
+- ...
```

Comment:
`- ...` looks like a placeholder for more information. Do you want to add more items to the list?

Reply:
I could, but there are so many options, that if I mention all, I would have written the docs for the AWS SDK :-) I thought this way I indicate, that the list is not complete. But we can remove the ... or replace it with anything else.

Reply:
```suggestion
```

Let's drop the placeholder and say it's a partial list in the text before the list items.